packages/vue/src/components/molecules/SfBottomModal/SfBottomModal.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="sf-bottom-modal">
+    <SfOverlay
+      :visible="isOpen"
+      class="sf-bottom-modal__overlay"
+      @click="close"
+    />
+    <transition name="sf-bottom-modal">
+      <nav
+        v-show="isOpen"
+        role="dialog"
+        aria-modal="true"
+        class="sf-bottom-modal__container"
+      >
+        <!--@slot Use this slot to replace title. -->
+        <slot name="title">
+          <SfHeading
+            v-if="title"
+            :level="3"
+            :title="title"
+            class="sf-bottom-modal__title"
+          />
+        </slot>
+        <!--@slot Use this slot to replace close button for desktop. -->
+        <slot name="close-desktop">
+          <SfCircleIcon
+            class="sf-circle-icon--small sf-bottom-modal__close-desktop desktop-only"
+            aria-label="Close"
+            icon="cross"
+            @click="close"
+          />
+        </slot>
+        <!--@slot Use this slot to replace content. -->
+        <slot />
+        <!--@slot Use this slot to replace close button for mobile. -->
+        <slot name="close-mobile">
+          <SfButton
+            class="sf-button--full-width mobile-only sf-bottom-modal__close"
+            aria-label="Close"
+            @click="close"
+            >Cancel</SfButton
+          >
+        </slot>
+      </nav>
+    </transition>
+  </div>
+</template>
+<script>
+import SfOverlay from "../../atoms/SfOverlay/SfOverlay.vue";
+import SfHeading from "../../atoms/SfHeading/SfHeading.vue";
+import SfButton from "../../atoms/SfButton/SfButton.vue";
+import SfCircleIcon from "../../atoms/SfCircleIcon/SfCircleIcon.vue";
+export default {
+  name: "SfBottomModal",
+  components: { SfOverlay, SfButton, SfCircleIcon, SfHeading },
+  props: {
+    /**
+     * Bottom Modal is open
+     */
+    isOpen: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * Bottom Modal title
+     */
+    title: {
+      type: String,
+      default: "",
+    },
+  },
+  watch: {
+    isOpen: {
+      handler(value) {
+        if (typeof window === "undefined" || typeof document === "undefined")
+          return;
```

Comment:
would be great to have some helper for that to reuse across the whole project, something like this
https://github.com/DivanteLtd/vue-storefront/blob/6cd57ab269a8a31ed8eb5a18ebc8934abe1b9b55/core/helpers/index.ts#L190

Reply:
sure, that's a good idea :)